tests/core/block/e2e/test_correctness.py

```diff
@@ -54,6 +43,11 @@ def test_v1_v2_greedy_equality_with_preemption(baseline_llm_generator,
 
     NOTE: We want a significant number of generated tokens so that any incorrect
     KV mapping has time to build up error.
+
```

Comment:
nit- Please consider updating the comment "Verify block manager v2 produces same outputs as block manager v1, even when there is preemption." to "Verify block manager produces same outputs even when there is preemption"

Reply:
Changed! TY for suggestion.